pkgs/development/python-modules/beets/default.nix

```
@@ -106,7 +106,14 @@
   runCommand,
 }:
 
-buildPythonPackage rec {
+let
+  # Avoid using `rec`, so that using e.g `passthru` or any other attributes
+  # defined inside, will have to be done via the beets argument, which can be
+  # overriden. Until `finalAttrs` support reaches `buildPythonPackage`, there
+  # is no way to avoid this.
```

```suggestion
  # overriden. Until `finalAttrs` support reaches `buildPythonPackage`, there
  # is no way to avoid this.
  #
  # See: https://github.com/NixOS/nixpkgs/issues/258246
```

Nice to have a link to the relevant issue here.